Modify the outdated descriptions related to parachutes (#7767)
ArduPilot/ardupilot_wiki: common/source/docs/common-parachute.rst

Connecting to the Autopilot
===========================

> The parachute release mechanism can be triggered from either a Relay or a PWM (i.e. Servo). If a Relay pin is used, use a release mechanism that requires a high output level to trigger, since GPIOs are forced low during the bootloader period. Set the corresponding ``RELAYx_DEFAULT`` to OFF or No Change to avoid triggering the release during autopilot initialization.

Reviewer: ```suggestion
The parachute release mechanism can be triggered from either a Relay or a PWM (i.e. Servo). If a Relay pin is used, use a release mechanism that requires a high output level to trigger, since GPIOs are forced low during the bootloader period, and set the corresponding ``RELAYx_DEFAULT`` to OFF or No Change to avoid triggering the release during autopilot initialization.
```

> The parachute release mechanism can be triggered from either a Relay or a PWM (i.e. Servo). If a Relay pin is used, use a release mechanism that requires a high output level to trigger, since GPIOs are forced low during the bootloader period. Set the corresponding ``RELAYx_DEFAULT`` to OFF or No Change to avoid triggering the release during autopilot initialization.

Reviewer 2: Am I reading correctly that alerting that HIGH must be used to prevent trigger-on-init is duplicated here and down below in the GPIO subsection?

If yes, I suggest a "Don't Repeat Yourself" pattern, so that all information is written in one spot (with links/references as needed). So one option is to simply remove this here (because it is clearly written below). Another option is to "reference" it, perhaps like this.
```suggestion
The parachute release mechanism can be triggered from either a Relay or a PWM (i.e. Servo). If a Relay pin is used, pay careful attention to the information in the GPIO subsection below to prevent an unsafe & unexpected release.
```

But this is "just my opinion". I happily defer to the author's as well as all reviewers' judgements if anyone else prefers keeping the info in here twice. No problem!

Reviewer: In general, I agree...but when it comes to safety (you do not want to be around an deploying (explosively) parachute,  I accept duplications

Reviewer 2: We are probably already on the same page, but just in case:
- I understand and support bending all ideals in the name of improved safety. That's top priority.
- My concern with duplications is the future risk that it changes in one spot, but not the other... then the duplication suddenly (and unexpectedly) switches from a safety protection to a safety problem.

IMO a suitably strongly worded "reference" gains most of the safety benefit with minimal risk of future de-sync.

Author: Thanks, I understand the DRY concern.

My preference is to keep the explicit warning here for now, since this is safety-related and it is useful for readers to see the relay/GPIO risk at this point in the page. I agree the detailed setup guidance should mainly live in the GPIO subsection below, and I’m happy to adjust if maintainers prefer the shorter reference-only wording.

Reviewer 2: (Thanks! I'm satisfied. Please resolve this comment whenever you like, I'm sadly unable to resolve my own comments.)

    :ref:`This function is a GPIO and has limited current capabilities.<gpio-warning>`

- Determine/Configure a pin to be a GPIO (see :ref:`common-gpios`).
- Set that pin number as one of the RELAY outputs and assign that relay to the parachute function. For example, for GPIO pin 51 using the first relay, set :ref:`RELAY1_PIN<RELAY1_PIN>` = 51 and :ref:`RELAY1_FUNCTION<RELAY1_FUNCTION>` = 3 (Parachute).
- Since GPIOs are always set low initially during the bootloader period, to avoid accidental release, always use a release mechanism that needs a high output level to trigger and set.

> - :ref:`CHUTE_TYPE<CHUTE_TYPE>` = 0 to release with a relay. Set the corresponding relay function parameter, such as :ref:`RELAY1_FUNCTION<RELAY1_FUNCTION>`, to 3 (Parachute) to choose which relay output is used.

Reviewer: ```suggestion
- :ref:`CHUTE_TYPE<CHUTE_TYPE>` = 0 to release with a relay. Set the corresponding relay function parameter, such as :ref:`RELAY1_FUNCTION<RELAY1_FUNCTION>` (for RELAY1), to 3 (Parachute) to choose which relay output is used.
```

RC Disable/Enable of Parachute
------------------------------

> You can disable or enable parachute release using an RC channel/switch: set an ``RCx_OPTION`` to 21. A high position enables parachute release; other positions disable it. If set to 23 (Parachute 3Pos), the low position disables, middle position enables operation, and a high will attempt to force release, as explained above, assuming the enabling conditions described below.

Reviewer: ```suggestion
You can disable or enable parachute release using an RC channel/switch: set an ``RCx_OPTION`` to 21. A high position enables parachute release; other positions disable it. If an ``RCx_OPTION`` is set to 23 (Parachute 3Pos), the low position disables parachute release, middle position enables operation, and a high will attempt to force release, as explained above, assuming the enabling conditions described below.
```
